e2e-tests/demand-backup/run

```diff
@@ -7,6 +7,44 @@ test_dir=$(realpath $(dirname $0))
 
 set_debug
 
+check_finalizer_for_fs() {
+  local backup_name="$1"
+  local finalizer_flag="off"
+
+  desc "Checking finalizers on backup $backup_name"
+  # Check if pxc-backup has the finalizer "percona.com/delete-backup"
+  if kubectl get pxc-backup "$backup_name" -o jsonpath='{.metadata.finalizers}' | grep -q "percona.com/delete-backup"; then
+    finalizer_flag="on"
+  fi
+
+  echo "Finalizer is $finalizer_flag"
+
+  echo "Looking for PVCs containing '$backup_name'"
+  pvc=$(kubectl get pvc --no-headers -o custom-columns=":metadata.name" | grep "$backup_name")
+
+  if [[ -z "$pvc" ]]; then
+    echo "No PVCs found with substring '$backup_name'"
```

Comment:
Extend error output to be more descriptive. Maybe smth like:  echo "No PVCs found with substring '$backup_name' before the backup deletion. Smth wrong with backup"?